tests/inputFields.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { test, expect } from "@playwright/test";
+
+test.beforeEach(async ({ page }) => {
+  await page.goto("/");
+  await page.getByTitle("pettypes").click();
+  await expect(page.locator("h2")).toHaveText("Pet Types");
+});
+
+test.describe("Interacting with Input Fields", async () => {
+  test("Update pet type", async ({ page }) => {
+    const editPetType = page.getByRole("textbox");
```

Comment:
Node descriptive enough constant name
what is `editPetType`? Is it a button, menu item, dropdown, header, toggle selector?
It is `editPetTypeInputField`

Reply:
done. Thanks!